src/main/java/com/skiwi/githubhooksechatservice/mvc/beans/tasks/CommentsScanTask.java

```diff
@@ -15,13 +17,13 @@
 import com.skiwi.githubhooksechatservice.stackapi.StackExchangeComment;
 
 public class CommentsScanTask implements Runnable {
-    private static final Logger logger = LogManager.getLogger(CommentsScanTask.class);
-    
-    private Instant nextFetch = Instant.now();
-    private long lastComment;
-    private long fromDate;
-    private int remainingQuota;
-    
+	private static final Logger logger = LogManager.getLogger(CommentsScanTask.class);
+	
+	private Instant nextFetch = Instant.now();
+	private long lastComment;
+	private long fromDate;
+	private int remainingQuota;
+	
```

Comment:
Cool whitespace-changes... I don't want to be pedantic and I don't want to come off as jerk, but I'd not like my formatting preferences shot by a PR. Then again neither @skiwi2 nor @Zomis provided any shared formatting preferences and I am just a random person on the internet ;)